src/VideoList.js

```diff
@@ -5,8 +5,7 @@ import VideoItem from './VideoItem'
 class VideoList extends React.Component {
 
   render() {
-    const donkey = this.props.musicVideos;
-    var currentArtistVideos = donkey;
+    const {musicVideo: currentArtistVideos } = this.props;
```

Comment:
Consider [destructuring properties](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) from the `this.props` object. Note how I aliased `musicVideo` to `currentArtistVideos` as well. Destructuring allows you to concisely reference items from a collection.

Also note: if you're initializing state in a class component - you can destructure any props you're initializing the component with in the `constructor` method ([click for docs](https://reactjs.org/docs/react-component.html#constructor)).